concent_api/verifier/utils.py

```diff
@@ -96,3 +97,46 @@ def delete_file(file_path):
             os.unlink(file_path)
     except OSError as exception:
         logger.warning(f'File with path {file_path} was not deleted, exception: {exception}')
+
+
+def upload_file_to_storage_cluster(file_content, file_path, upload_token):
+    dumped_upload_token = dump(upload_token, None, settings.CONCENT_PUBLIC_KEY)
+    b64_encoded_token = b64encode(dumped_upload_token).decode()
+    headers = {
+        'Authorization': 'Golem ' + b64_encoded_token,
+        'Concent-Auth': b64encode(
```

Comment:
why `b64encode(<SOMETHING>).decode()`?

Reply:
We do this in the same way in `request_upload_status` in `transer_operations.py` and in cluster tests, so I think it is proper way to do it.

Reply:
@dybi It's Ok because it's `bytes.decode()`. It's just a conversion from `bytes` to `str`.

It's equivalent to `bytes.decode('utf-8')`. i.e. it takes a sequence of bytes, assumes it has UTF-8 encoding and returns a `str`. UTF-8 is not really relevant here. We could do `bytes.decode('ascii')` just as well and it would work too, because `b64encode()` is garanteed to include only characters from a narrow subset of ASCII.

As an aside, I'd prefer if we always passed a parameter to `decode()` explicitly. It would make it easier to see what we are doing and that we decode as 'utf-8'. Also, `decode('ascii')` would really be more appropriate here because we don't expect any unicode characters anyway.